rails-template.rb

```diff
@@ -0,0 +1,596 @@
+      },
+    },
+  };
+JS
+
+file "app/javascript/application.ts", <<~JS
+  import '@hotwired/turbo-rails';
+  import { start } from '@nerdgeschoss/shimmer';
+  import { application } from 'controllers/application';
+  import './controllers';
+
+  start({ application });
+JS
+
+file "app/javascript/controllers/application.ts", <<~JS
+  import { Application } from '@hotwired/stimulus';
+
+  const application = Application.start();
+
+  application.debug = false;
+
+  export { application };
+JS
+
+initializer "i18n.rb", <<~RUBY
+  # frozen_string_literal: true
+
+  I18n.default_locale = :en
+  disabled_locales = ENV["DISABLED_LOCALES"].to_s.split(",").map(&:downcase).map(&:to_sym)
+  I18n.available_locales = [:en] - disabled_locales
+  I18n.enforce_available_locales = true
+
+  Rails.application.configure do
+    config.i18n.fallbacks = [:en] - disabled_locales
+  end
+RUBY
+
+initializer "oj.rb", <<~RUBY
+  # frozen_string_literal: true
+
+  Oj.optimize_rails
+RUBY
+
+initializer "serviceworker.rb", <<~RUBY
+  # frozen_string_literal: true
+
+  Rails.application.configure do
+    config.serviceworker.routes.draw do
+      match "/serviceworker.js"
+    end
+  end
+RUBY
+
+initializer "uuid.rb", <<~RUBY
+  # frozen_string_literal: true
+
+  Rails.application.configure do
+    config.generators do |g|
+      g.orm :active_record, primary_key_type: :uuid
+    end
+  end
+RUBY
+
+file "app/views/layouts/application.html.slim", <<~SLIM
+  doctype html
+  html lang=I18n.locale
+    = render "components/head"
+    body class=Rails.env
+      = yield
+SLIM
+
+file "app/views/components/_head.html.slim", <<~SLIM
+  head
+    meta charset="UTF-8"
+
+    / WebApp
+    meta name="viewport" content="width=device-width, initial-scale = 1.0, viewport-fit=cover"
+    / link rel="manifest" href=manifest_path
+    / link rel="apple-touch-icon" href=asset_path("app-icon.png")
+    meta name="theme-color" content="#1d2024"
+    / SEO
+    /= favicon_link_tag asset_path('favicon.svg')
+
+    / Security
+    = csrf_meta_tags
+    = csp_meta_tag
+
+    / Assets
+    = stylesheet_link_tag "application", media: "all", "data-turbo-track": "reload"
+    = javascript_include_tag "application", "data-turbo-track": "reload", defer: true
+
+    = yield :additional_head_tags if content_for? :additional_head_tags
+SLIM
+
+file "app/controllers/application_controller.rb", <<~RUBY
+  # frozen_string_literal: true
+
+  class ApplicationController < ActionController::Base
+    include Shimmer::Localizable
+    include Shimmer::RemoteNavigation
+
+    before_action :check_locale
+  end
+RUBY
+
+file "app/controllers/pages_controller.rb", <<~RUBY
+  # frozen_string_literal: true
+
+  class PagesController < ApplicationController
+    def home
+    end
+  end
+RUBY
+
+file "app/views/pages/home.html.slim", <<~SLIM
+  h1 #{app_name}
+SLIM
+
+file ".rspec", "--require spec_helper"
```

Comment:
@JensRavens : Confused here. You debate for letting go of _RSpec_ for so long. And now, it's part of our _Rails_ template for new projects?